packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -62,6 +66,27 @@ def noop(b, opts):
 }
 
 
+def _apply_sysctl(setting, service):
+    try:
+        check_call(["sysctl", "-q", "-w", setting])
+    except CalledProcessError:
+        log.warning("sysctl {setting} not set for {service}".format(setting=setting, service=service))
+
+
+def _apply_sysctl_settings(sysctl_settings, service):
+    for setting, value in sysctl_settings.get(service, {}).items():
+        _apply_sysctl("{setting}={value}".format(setting=setting, value=value), service)
+
+
+def _apply_service_configuration(service):
+    if not os.path.exists(_DCOS_SERVICE_CONFIGURATION_JSON):
```

Comment:
nit: Ideally this code would probably be in pkgpanda, and bootstrap could call out to it (Would also make it easier to share in open vs. downstream bootstrap implementations). Currently downstream bootstrap will have to re-implement it from scratch.